src/server/scripts/Northrend/IcecrownCitadel/boss_lady_deathwhisper.cpp

```diff
@@ -247,9 +271,9 @@ class boss_lady_deathwhisper : public CreatureScript
                 events.ScheduleEvent(EVENT_SPELL_DEATH_AND_DECAY, 10000);
                 if (GetDifficulty() != RAID_DIFFICULTY_10MAN_NORMAL)
                     events.ScheduleEvent(EVENT_SPELL_DOMINATE_MIND_25, 27000);
-                events.ScheduleEvent(EVENT_SPELL_SHADOW_BOLT, urand(5500, 6000), 0, PHASE_ONE);
+                events.ScheduleEvent(EVENT_SPELL_SHADOW_BOLT, 2000, 0, PHASE_ONE);
```

Comment:
Because this event"EVENT_SPELL_SHADOW_BOLT" is too dense, the summoning event"EVENT_SUMMON_WAVE_P1" is delayed

Reply:
If this event“EVENT_SPELL_SHADOW_BOLT” is fixed, the start time of the summoning event“EVENT_SUMMON_WAVE_P1” will be 7000，DBM is like this <7000>

Reply:
All BOSS events have this problem